packages/yourloops/pages/hcp/team-members-table.tsx

```diff
@@ -69,23 +69,21 @@ const teamMembersStyles = makeStyles((theme: Theme) => {
   };
 });
 
-function PersonRemoveIcon(): JSX.Element {
+function PersonRemoveIcon(props: SvgIconProps): JSX.Element {
```

Comment:
same comment as `VerifiedIcon`in team-card

Reply:
Will we fix in another PR